backend/tournesol/models/entity_poll_rating.py

```diff
@@ -67,4 +74,18 @@ def update_n_ratings(self):
             .distinct("user")
             .count()
         )
-        self.save(update_fields=["n_comparisons", "n_contributors"])
+
+        self.save(update_fields=["n_comparisons", "n_contributors", "sum_trust_scores"])
+
+    @staticmethod
+    def bulk_update_sum_trust_scores(poll):
+        EntityPollRating.objects.filter(poll=poll).update(
```

Comment:
Do we want to use the Django's method `bulk_update` here to benefit from the `batch_size` attribute?

We had some problems before with SQL queries locking tables for too long while updating all rows with `update`.

Reply:
More details on the differences between `update` and `bulk_update`:
- https://docs.djangoproject.com/en/4.1/ref/models/querysets/#bulk-update

Reply:
I made the two options available and tested

Reply:
Looks good!

I was wondering, do we really want to maintain the two options in our code base? Or are they temporarily here so we can profile the two options?

It could be good to test (if it has not been done already) the the `update` and the `bulk_update` on staging or a local machine with a dump of the production database. If we are confident that we can update thousands and thousands of  `EntityPollRating` in one query, I still think a priori that `bulk_update` is a better alternative. Using batch size of X is like saying "we don't know how our database server will react if we try to update more than X rows at the same time". And currently I don't know how our database will react if we update our 36 000 ratings in a single query.